src/shared/useMaxContentWidth.ts

```diff
@@ -0,0 +1,10 @@
+import {useTheme} from '@shopify/restyle';
+
+import {useOrientation} from './useOrientation';
+import {Theme} from './theme';
+
+export const useMaxContentWidth = (): number | undefined => {
```

Comment:
Rationale for this hook is that if we later want to change how the maxWidth is determined (say, maybe  percentage), we can just make the modification here.